pkg/apis/rollouts/v1alpha1/generated.proto

```diff
@@ -947,6 +947,8 @@ message KayentaMetric {
   optional KayentaThreshold threshold = 7 [(gogoproto.customname) = "Threshold", (gogoproto.nullable) = false];
 
   repeated KayentaScope scopes = 8 [(gogoproto.customname) = "Scopes", (gogoproto.nullable) = false];
+
+  optional bool lookback = 9 [(gogoproto.customname) = "Lookback", (gogoproto.nullable) = false];
```

Comment:
Hmmm, when I deleted this file, and reran `make codegen` against your branch , it had removed all the `gogoproto` annotations. I'm trying to understand why yours still generates this file with the annotations still in.

Reply:
I'd like to get the codegen fix in https://github.com/argoproj/argo-rollouts/pull/4482, and have this change rebase on top of it, so we can make sure the CI generates the file the same way.